vidarrtest-regression.json.in

```diff
@@ -998,7 +998,7 @@
             {
                 "metrics_calculate": "@CHECKOUT@/tests/calculate.sh",
                 "metrics_compare": "@CHECKOUT@/tests/compare.sh",
-                "output_metrics": "/.mounts/labs/gsi/testdata/bamqc/output_expectation/5.1.1/bamQC_test_multiple_bams_input.metrics",
+                "output_metrics": "/.mounts/labs/gsi/testdata/bamqc/output_metrics/workflow_test_06.metrics",
```

Comment:
I would have keep it, especially that for bamQC we have at least a couple of different versions. This may be (there is a very slight chance) useful in a future if we decide to maintain separate implementations